packages/page-staking/src/Performance/CurrentList.tsx

```diff
@@ -0,0 +1,198 @@
+        return [undefined, undefined, undefined];
+      }
+
+      const erasStartSessionIndexLookup: Record<number, number> = {};
+
+      erasStartSessionIndex.filter(([, values]) => values.isSome)
+        .forEach(([key, values]) => {
+          const eraIndex = key.args[0];
+
+          erasStartSessionIndexLookup[eraIndex.toNumber()] = values.unwrap().toNumber();
+        });
+
+      let currentEra = sessionInfo.activeEra.toNumber();
+      let currentEraSessionStart: number = erasStartSessionIndexLookup[currentEra];
+      let currentEraSessionEnd = null;
+
+      while (currentEraSessionStart > session) {
+        currentEraSessionEnd = currentEraSessionStart - 1;
+        currentEra = currentEra - 1;
+        currentEraSessionStart = erasStartSessionIndexLookup[currentEra];
+      }
+
+      return [currentEra, currentEraSessionStart, currentEraSessionEnd];
+    },
+    [erasStartSessionIndex, sessionInfo, session]
+  );
+
+  const blocksTarget = useMemo(
+    () => {
+      if (!committeeSize || ! sessionInfo) {
+        return undefined;
+      }
+      const sessionPeriod = api.consts.elections.sessionPeriod;
+
+      const currentCommitteeSize = committeeSize.nonReservedSeats.toNumber() + committeeSize.reservedSeats.toNumber();
+      return Math.round(Number(sessionPeriod.toString()) / currentCommitteeSize);
+    },
+     [committeeSize, sessionInfo, api]
+  );
+
+  const isLoading = useLoadingDelay();
+
+  const { validators } = useMemo(
+    () => getFiltered(targets, favorites, currentSessionCommittee),
+    [favorites, targets, currentSessionCommittee]
+  );
+
+  const list = useMemo(
+    () => isLoading
+      ? undefined
+      : validators,
+    [isLoading, validators]
+  );
+
+  const infoMap = useMemo(
+    () => targets.validators && mapValidators(targets.validators),
+    [targets]
+  );
+
+  const headerRef = useRef(
+    [
+      [t('validators'), 'start', 2],
+      [t('blocks created'), 'expand'],
+      [t('blocks expected'), 'media--1100'],
+      [t('max % reward')],
+      [],
+      [undefined, 'media--1200']
+    ]
+  );
+
+  return (
+    <Table
+      className={className}
+      empty={
+        list && infoMap && t<string>('No active validators found')
+      }
+      emptySpinner={
+        <>
+          {!validators && <div>{t<string>('Retrieving validators')}</div>}
+          {!infoMap && <div>{t<string>('Retrieving validator info')}</div>}
+          {!list && <div>{t<string>('Preparing validator list')}</div>}
+        </>
+      }
+      filter={
+        <Filtering
+          nameFilter={nameFilter}
+          setNameFilter={setNameFilter}
+        />
+      }
+      header={headerRef.current}
+    >
+      {list && list.map(([address, isFavorite]): React.ReactNode => (
+        <Address
+          address={address}
+          blocksCreated={sessionValidatorBlockCountLookup ? sessionValidatorBlockCountLookup[address] : 0}
+          blocksTarget={blocksTarget ? blocksTarget : 0}
```

Comment:
if the validator is not in the committee this should be equal zero

Reply:
Good point, goal of this new page was to show the committee for a given session, not the era. In this PR we just show it for the current session, and in the subsequent PRs, let me fix that in the next one.